### PR TITLE
Project Curation

### PR DESCRIPTION
**Pull request by JordoBrooks**
Basic initial implementation of project curation

### CODE REVIEW

File: smart-contracts/ethereum/contracts/Project.sol [hunk 1]
```diff
@@ -451,5 +474,26 @@ contract Project {
...
     }
 
+    function curate(bool _vote) public curatorOnly {
+        require(status == Status.Pending, "This project is not seeking curator approval at this time.");
+        require(draftCuration.isActive, "This project is not open for curation.");
+
+        _vote ? draftCuration.approvalCount++ : draftCuration.approvalCount--;
+    }
+
+    function publishProject() public devRestricted {
```
Comment (ericmuyser) [inline smart-contracts/ethereum/contracts/Project.sol]: Can this return the status so the frontend can check it was successfully published?

Comment (JordoBrooks) [reply smart-contracts/ethereum/contracts/Project.sol]: I've now made it fire an event with the updated status

**Comment by ericmuyser** [comment]
What's the status on this?

**Comment by JordoBrooks** [comment]
On hold until #24 is finished since it's going to require some refactoring to be compatible.

**Comment by JordoBrooks** [comment]
Curation functionality has been incorporated into #26

**Pull request closed without merging**